README.md

```diff
@@ -623,6 +625,116 @@ class PaymentCaptureViewModel @Inject constructor(
 It is the `funding_type` on a [Payment object](#step-0-create-a-payment-object)
 where you indicate whether a `Payment` will capture (SNAP or EBT Cash).
 
+## Collecting the PIN for a deferred EBT payment
+
+### Step 0: Create a `Payment` object
+
+Send a `POST` request to Forage's `/payments/` endpoint [to create a
+`Payment` object](https://docs.joinforage.app/reference/create-a-payment).
+You need the `ref` of the `Payment` within the `/payments` response for Step 3.
+
+### Step 1: Add the `ForagePINEditText` UI component
+
+As with [performing a balance check](#performing-a-balance-check), you will need
+to add a `ForagePINEditText` to your UI. Please reference
+[Step 1 of Performing a Balance Check](#step-1-add-the-foragepinedittext-ui-component)
+
+### Step 2: Customizing `ForagePINEditText`
+
+Please reference [Step 2 of Performing a Balance Check](#step-2-customizing-foragepinedittext)
+
+### Step 3: Call `setForageConfig()`
+
+Please reference [Step 3 of Performing a Balance Check](#step-3-call-setforageconfig)
+
+### Step 4: Collect the pin for the EBT payment
+
+The ForageSDK provides a method to collect a customer's PIN for an EBT payment and defer the capture of the payment to the server.
+
+```kotlin
+data class CollectPinParams(
```

Comment:
Thoughts on naming the params `DeferredCaptureParams`? Can do another name but `CollectPinParams` still throws me off a bit. Lmk what you think though
```suggestion
data class DeferredPaymentCaptureParams(
```